<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m1act1"/>
    <addaction name="m1act2"/>
    <addaction name="separator"/>
    <addaction name="m1act3"/>
    <addaction name="m1act4"/>
    <addaction name="separator"/>
    <addaction name="m1act5"/>
    <addaction name="separator"/>
    <addaction name="m1act6"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="m2act1"/>
    <addaction name="m2act2"/>
    <addaction name="separator"/>
    <addaction name="m2act3"/>
    <addaction name="m2act4"/>
    <addaction name="m2act5"/>
    <addaction name="m2act6"/>
    <addaction name="separator"/>
    <addaction name="m2act7"/>
    <addaction name="m2act8"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="m3act1"/>
    <addaction name="m3act2"/>
    <addaction name="separator"/>
    <addaction name="m3act3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="m4act1"/>
    <addaction name="m4act2"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="m5act1_2"/>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="m1act2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="m1act3">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="m1act4">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="m1act1">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="m1act5">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="m1act6">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="m2act1">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="m2act2">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="m2act4">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="m2act5">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="m2act3">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="m2act6">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="m2act7">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="m2act8">
   <property name="text">
    <string>Find and Replace</string>
   </property>
  </action>
  <action name="m3act3">
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="m4act1">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="m4act2">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="m4act3">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="m6act1">
   <property name="text">
    <string>About Editor</string>
   </property>
  </action>
  <action name="m3act1">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="m3act2">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="m5act1">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="m5act1_2">
   <property name="text">
    <string>About Editor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
